packages/dashboard/src/components/dashboard/index.test.tsx

```diff
@@ -20,6 +20,9 @@ it('renders', function () {
         },
         widgets: [],
         viewport: { duration: '5m' },
+        querySettings: {
```

Comment:
is it necessary to provide the `querySettings.refreshRate`? it should default to 5s without it